src/views/properties/propertiesFormStructure.jsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
```

Comment:
I think we could rename all these FormStructure files to something like TableComponents since there aren't forms on the pages

Reply:
I just renamed all of the three that we are able to use! Unfortunately the tickets page we can't do this for, because the definitions on that page require additional parameters for the unique link wrapper being used on that page..